dash-pipeline/SAI/sai_api_gen.py

```diff
@@ -136,6 +138,11 @@ def extract_action_data(program):
                 param[NAME_TAG] = p[NAME_TAG]
                 param['type'], param['field'] = get_sai_key_type(int(p[BITWIDTH_TAG]), p[NAME_TAG], p[NAME_TAG])
                 param['bitwidth'] = p[BITWIDTH_TAG]
+                if param['type'] == 'sai_object_id_t':
+                    if param[NAME_TAG].endswith('dash_acl_group_id'):
```

Comment:
There shouldn't be a reference to a table from DASH pipeline. It should be deducted from the list of tables after the API dictionary is fully formed by going over the tables once again, kind of like that:
```
for param in params:
    for table in tables:
        if param[NAME_TAG].endswith(table.[NAME_TAG]):
            param[OBJECT_NAME_TAG] = table[NAME_TAG][:-len("_id")]
```

Reply:
Fixed